.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'type: bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
More 'realistic' sample perhaps, also preferably using pjsua app, e.g:
1. run pjsua app with parameter: `--video --registrar sip:some.sip.registrar ...`
1. make two video calls to: `sip:abc` and `sip:xyz`1
1. setup video conference: `vid conf cc ...`
1. assertion occurs!